packages-content-model/roosterjs-content-model-api/lib/publicApi/utils/formatSegmentWithContentModel.ts

```diff
@@ -71,7 +68,7 @@ export function formatSegmentWithContentModel(
 
             afterFormatCallback?.(model);
 
-            if (!pendingFormat && isCollapsedSelection) {
+            if (isCollapsedSelection) {
```

Comment:
Could we merge this `if` statement into the next one? since they have the same condition.